### PR TITLE
Consolidate media queries to 2 standard max-widths

### PR DESCRIPTION
**Pull request by SYwaves**
Condenses the various media queries previously optimized for a series of screen size ranges into two: (max-width: 767px) and (max-width: 479px). Other queries have been moved to appropriate locations as well.

This is to reduce code complexity and better maintainability.

### CODE REVIEW

File: sigma9.css [hunk 1]
```diff
@@ -456,6 +458,83 @@ sup {
 	padding: 0 5px;
 }
 
+/* off-canvas */
+.close-menu {
+	display: none;
+}
```
Comment (ammongit) [inline sigma9.css:464]: What is `.close-menu` here? It doesn't appear to be in a `@media` query.

File: sigma9.css [hunk 2]
```diff
@@ -940,13 +1019,15 @@ div.curved {
...
 @media (max-width: 479px) {
+	blockquote,
 	div.blockquote,
 	div.curved {
 		margin: 1em 0;
 	}
 }
 
 @media (min-width: 480px) and (max-width: 580px) {
+	blockquote,
```
Comment (ammongit) [inline sigma9.css:1030]: :+1:

**Review by ammongit — APPROVED** [review]
Seems fine to me, make sure you also get a view from @stormbreath and @rossjrw

**Comment by rossjrw** [comment]
Test page for f3e8e3e: http://scptestwiki.wikidot.com/notheme:s9pr49 rev 0

File: sigma9.css (hunk 1 above)
Comment (rossjrw) [reply sigma9.css:464]: It's moved here from line 1334, randomly chucked between two media queries. This is a better place

**Review by rossjrw — APPROVED** [review]
Practically identical to current Sigma-9, LGTM. Themes that are tightly coupled to the current header positioning ([Anderson Robotics](https://scp-wiki.wikidot.com/theme:ar) comes to mind) will need to be updated to account.

Just needs a stylelint pass and it's good to go

**Review by stormbreath — APPROVED** [review]

**Pull request merged**